PARTagPicker/ViewController.m

```diff
@@ -29,8 +30,38 @@ - (void)viewDidLoad {
 
 - (void)initDummyData {
     //Data for demo project
-    self.allTags = @[@"one fish", @"two fish", @"red fish", @"blue fish", @"the cat in the hat", @"Seuss"];
-    self.preChosenTags = @[@"in a box", @"with a fox", @"thing 1", @"thing 2"];
+    /*self.allTags = @[@"one fish", @"two fish", @"red fish", @"blue fish", @"the cat in the hat", @"Seuss"];
+    self.preChosenTags = @[@"in a box", @"with a fox", @"thing 1", @"thing 2"];*/
+    
+    NSArray *allTags = @[@"one fish", @"two fish", @"red fish", @"blue fish", @"the cat in the hat", @"Seuss"];
+    NSArray *preChosenTags = @[@"in a box", @"with a fox", @"thing 1", @"thing 2"];
+    
+    NSMutableArray<PARTag *> *allTagsCreated = [[NSMutableArray alloc] initWithCapacity:allTags.count];
+    NSMutableArray<PARTag *> *allTagsPreChosen = [[NSMutableArray alloc] initWithCapacity:preChosenTags.count];
+    
+    __weak ViewController *weakSelf = self;
```

Comment:
I left it because of the code inside the block, but indeed that should be commented also.